File: nnamesarray.c
// read and print n marks using array

#include <stdio.h>
int main()
{
    int i, countOfNames;  
    printf("Enter the number of students's name to be printed: ");
    scanf("%d", &countOfNames);
    char namesArray[countOfNames];
    for(i = 0; i < countOfNames; i++)
    {
        printf("Enter the name of student %d: ", i + 1);
        scanf("%s", &namesArray[i]);
    }
    printf("\nThe names of the students are. \n");
    for(i = 0; i < countOfNames; i++)
    {
        printf("The student %d name: ", i + 1);
        printf("%s \n", namesArray[i]);
    }
    return 0;
}

File: Python/nnamesarray.py
# read and print n names using array

countOfNames = int(input("Enter the number of students's name to be printed: "))
namesArray = []
for counter in range(0, countOfNames):
	names = str(input("Enter the name of the student %d: " %(counter + 1)))
	namesArray.append(names)
print("\nThe names of students are: ", namesArray)